src/jquery.multiselect.js

```diff
@@ -113,66 +123,67 @@
 
       // The button that opens the widget menu.  Note that this is inserted later below.
       var $button = (this.$button = $( document.createElement('button') ) )
-           .addClass('ui-multiselect ui-widget ui-state-default ui-corner-all' + (classes ? ' ' + classes : ''))
-           .attr({ 
-                  'type': 'button', 
-                  'title': elSelect.title, 
-                  'tabIndex': elSelect.tabIndex, 
-                  'id': elSelect.id ? elSelect.id  + '_ms' : null 
-                })
-           .prop('aria-haspopup', true)
-           .html('<span class="ui-multiselect-open">' + iconSet.open + '</span>');   // Necessary to simplify dynamically changing the open icon.
-
-        this.$buttonlabel = $( document.createElement('span') )
+            .addClass('ui-multiselect ui-widget ui-state-default ui-corner-all' + (classes ? ' ' + classes : ''))
+            .attr({ 'type': 'button', 'title': elSelect.title, 'tabIndex': elSelect.tabIndex, 'id': elSelect.id ? elSelect.id  + '_ms' : null })
```

Comment:
I would prefer you expanded it again, easier to read with the ternary hidden in there.

Reply:
Will do.